blogs/converter.js

```diff
@@ -0,0 +1,20 @@
+const {readdirSync} = require('fs');
+const {readFileSync} = require('fs');
+const {writeFileSync} = require('fs');
+
+const {marked} = require('marked');
+
+const listOfMarkdownFiles = readdirSync('markdown');
+
+console.log(listOfMarkdownFiles);
+
+listOfMarkdownFiles.forEach(function(element){
```

Comment:
you can do this in the new way with arrow functions

```
listOfMarkdownFiles.forEach(element => {
...
})
```
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions

Reply:
So that's a variable? You can make variables without even using var or const?

Reply:
so youre not declaring a variable, its a value being return by the method `forEach`